src/cartservice/cartservice.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Not a big deal, but why did you have to update to 3.1?

Reply:
Originally, I was trying using that using syntax without the {} and that needed a version bump. Now we need it because I updated to use that syntax.